content/200-concepts/100-components/drivers.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Drivers'
+metaTitle: 'Drivers'
+metaDescription: 'Learn how Prisma connects to your database using the built-in drivers and how you can use Prisma along with other JavaScript database drivers using driver adapters (Preview)'
+preview: true
+tocDepth: 4
+---
+
+## Built-in drivers (default)
+
+One of Prisma Client's component's is the [Query Engine](./prisma-engines/query-engine) <span class="concept"></span>. The Query Engine is responsible for transforming Prisma Client queries to SQL statements. The Query Engine connects to your database using the included drivers that don't require additional setup. The built-in drivers use TCP connections to connect to the database.
+
+![Query flow from the user application to the database with Prisma Client](./images/drivers/qe-query-execution-flow.png)
+
+## Driver adapters
+
+Prisma Client can connect and run queries against your database using other JavaScript database drivers using **driver adapters**. Adapters act as _translators_ between Prisma Client and the JavaScript database driver.
+
+![Query flow from the user application to the database using Prisma Client and driver adapters](./images//drivers/qe-query-engine-adapter.png)
+
+Prisma will use the Query Engine to transform the Prisma Client query to SQL and run the generated SQL queries via the JavaScript database driver.
+
+Prisma maintains the following driver adapters:
+
+- Serverless database driver adapters:
+  - [Neon](#neon)
+  - [PlanetScale](#planetscale)
+- Database driver adapters:
+  - [Turso](#turso)
+
+To use this feature:
+
+1. Update the `previewFeatures` block in your schema to include the the `driverAdapters` preview feature:
+
+   ```prisma
+   generator client {
+     provider        = "prisma-client-js"
+     previewFeatures = ["driverAdapters"]
+   }
+   ```
+
+2. Generate Prisma Client:
+
+   ```sh
+   npx prisma generate
+   ```
+
+### Serverless drivers
+
+By default, JavaScript database drivers use TCP connections. Several database providers enable use of different protocols, such as HTTP and WebSocket, to connect to your database. Example database providers include Neon and PlanetScale.
```

Comment:
> By default, JavaScript database drivers use TCP connections
Is this really what we want to highlight here? Why?

Reply:
We don't have to. Happy to do away with it.